test/index-test.ts

```diff
@@ -76,11 +78,12 @@ describe('hold', () => {
       }
     }
 
-    const test = (source, expected) => {
+    const test = <A>(source: Stream<A>, expected: A[]) => {
```

Comment:
Return type here seems to be `Promise<readonly A[]>`, although the tests don't really care, so `Promise<unknown>` is probably enough.

Reply:
Fixed